apps/heureka/src/App.test.jsx

```diff
@@ -4,17 +4,21 @@
  */
 
 import React from "react"
-import { render } from "@testing-library/react"
-
+import { render, act, waitFor } from "@testing-library/react"
 // support shadow dom queries
 // https://reactjsexample.com/an-extension-of-dom-testing-library-to-provide-hooks-into-the-shadow-dom/
 import { screen } from "shadow-dom-testing-library"
 import App from "./App"
 
 describe("App", () => {
   test("should render the App component", async () => {
-    render(<App id="123" />)
-    const loginTitle = screen.queryAllByShadowText(/HEUREKA/i)
-    expect(loginTitle.length > 0).toBe(true)
+    await act(async () => {
```

Comment:
Why you do here a render under the act it is usually it is only used on state change or any side effects(like clicks, etc)

Reply:
To solve the following warning, I used act. If not, what is your suggestion?
![image](https://github.com/user-attachments/assets/2631efae-fa5f-4025-b5e6-7ec67863f399)